model_api/cpp/cmake/model_apiConfig.cmake

```diff
@@ -1,5 +1,5 @@
 include(CMakeFindDependencyMacro)
-find_dependency(OpenCV)
-find_dependency(OpenVINO)
+find_package(OpenCV REQUIRED COMPONENTS core imgproc)
+find_package(OpenVINO REQUIRED COMPONENTS Runtime)
```

Comment:
`find_dependency` is suitable here, otherwise:
```
find_package(model_api QUIET)
```
can result in `FATAL_ERROR` if either OpenCV or OpenVINO is not available